includes/admin/class-wcmp-export-consignments.php

```diff
@@ -471,4 +471,35 @@ private function setBaseData(): void
             ->setDeliveryType($this->getDeliveryType())
             ->setLabelDescription($this->getFormattedLabelDescription());
     }
+
+    /**
+     * @throws Exception
+     */
+    public function validate(): bool
+    {
+        if (! $this->consignment->validate()) {
+            return false;
+        }
+
+        $weight         = $this->getTotalWeight(); // NOTE this is already the weight per consignment (in grammes)
```

Comment:
We doen eigenlijk geen end of line comments, dit mag je erboven zetten. Ik zie ook niet echt waarom dit erbij moet staan.

Reply:
Ik dacht in het begin dat getTotalWeight de Total weight zou teruggeven gezien de naam, maar het gaf de weight per collo terug, vandaar de reminder, maar ik zal de comment verwijderen